src/main/java/org/jenkinsci/test/acceptance/recorder/HarRecorder.java

```diff
@@ -0,0 +1,58 @@
+package org.jenkinsci.test.acceptance.recorder;
+
+import net.lightbody.bmp.BrowserMobProxy;
+import net.lightbody.bmp.BrowserMobProxyServer;
+import net.lightbody.bmp.core.har.Har;
+import net.lightbody.bmp.proxy.CaptureType;
+import org.jenkinsci.test.acceptance.junit.FailureDiagnostics;
+import org.jenkinsci.test.acceptance.junit.GlobalRule;
+import org.jenkinsci.test.acceptance.utils.SystemEnvironmentVariables;
+import org.junit.rules.TestWatcher;
+import org.junit.runner.Description;
+
+import javax.inject.Inject;
+import java.io.File;
+import java.io.IOException;
+
+@GlobalRule
+public class HarRecorder extends TestWatcher {
+    static String CAPTURE_HAR = SystemEnvironmentVariables.getPropertyVariableOrEnvironment("HAR", null);
```

Comment:
Ought to be explained in some `docs/*.md`.

Reply:
Good point, I'll find the right place in the docs to put it.